NEWS.md

```diff
@@ -21,6 +21,12 @@ BREAKING CHANGES AND DEPRECATIONS
   multiple tables is returned. Furthermore, `print_html()` did not work, which
   was also fixed now.
 
+* `demean()` (and `degroup()`) gets an `append` argument that defaults to `TRUE`,
```

Comment:
This will require also updating the examples across easystats:

- the `demean` vignette in `{parameters}`
- `parameters:::.std_info_pseudo()`
- tests in `{parameters}`

Reply:
Right, but the datawizard update will break parameters anyway